Tests/Security/LiipImagineThumbnailTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Security;
+
+use Gaufrette\Adapter\InMemory;
+use Gaufrette\File;
+use Gaufrette\Filesystem;
+use Sonata\MediaBundle\Tests\Helpers\PHPUnit_Framework_TestCase;
+use Sonata\MediaBundle\Thumbnail\LiipImagineThumbnail;
+
+class LiipImagineThumbnailTest extends PHPUnit_Framework_TestCase
```

Comment:
We no longer extend `Sonata\MediaBundle\Tests\Helpers\PHPUnit_Framework_TestCase` in tests, please have a look at the other test cases